source/base/parameter_handler.cc

```diff
@@ -2386,7 +2386,7 @@ ParameterHandler::recursively_print_parameters (const std::vector<std::string> &
             // the documentation, and then the actual entry; break the
             // documentation into readable chunks such that the whole
             // thing is at most 78 characters wide
-            if ((!(style & 128)) &&
+            if ((style == Text) &&
```

Comment:
Shouldn't this be `style != ShortText`? At least `ShortText` is the only enum that has the 128-bit set.
In particular, in line 2615 `!(style & 128)` is replaced by `style != ShortText` as well.